style.css

```diff
@@ -0,0 +1,546 @@
+body{
+    width: 428px;
```

Comment:
Šita taisyklė neleidžia aplikacijai būtų didesnei nei 428px dėl to didesniuose įrenginiuose atrodo prastai